index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="author" content="Dagmawi Yohannes" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>portfolio</title>
+        <meta name="title" content="mobile first" />
+        <meta name="description" content="mobile first web design" />
+        <link rel="icon" type="image/png" sizes="32x32" href="images/logo.png" />
+        <link rel="stylesheet" href="css/styles.css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <section class="main">
+          <div>
+            <!-- navbar setup -->
+            <header>
+                <nav>
+                    <ul class="menu">
+                      <li>Portfolio</li>
+                      <li>About</li>
+                      <li>Contact</li>
+                    </ul>
+                  </nav>
+            </header>
+            <!-- main body -->
+        <main class="body">
+            <div>
+              <section>
+                <br />
+                <h1>Hi, I'm Dagmawi</h1>
+                <h2>I’m a software Developer</h2>
+                <br />
+              </section>
+              <section>
+                <p>
+                    My name is Dagmawi Yohannes I'm a Front End Engineer based inAddis Ababa, Ethiopia ☀️.
+                    I describe myself as a passionate developer who loves coding, open source, and 
+                    the web platform
+                </p>
+                <br />
+              </section>
+              <section>
+                <div class="h2">
+                    <span class="h2-text">LET'S CONNECT</span>
```

Comment:
- [ ] Please remove this `span` element because it is not a part of template 3 of the Figma design

Reply:
Thanks for all comments and appreciate your time...

Reply:
Your welcome